consensus/scp/tests/mock_network/mod.rs

```diff
@@ -1,21 +1,21 @@
 // Copyright (c) 2018-2020 MobileCoin Inc.
-#![allow(unused_attributes)]
-//TODO -- which attribute is unused???
+
+// The separated integration tests do not use all common code.
+// TODO: consider breaking out new_mesh and new_cyclic into separate integration code source.
+#![allow(dead_code)]
```

Comment:
Whats being unused?

Reply:
the mesh tests don't use `new_cyclic()` and the cyclic tests don't use `new_mesh()`... I have a fix ready to go but wanted to keep this PR as small as possible.